test/unit/tasks/MilestoneSpec.scala

```diff
@@ -60,5 +60,19 @@ class MilestoneSpec extends WordSpec with Matchers with TryValues {
 
       report.close()
     }
+
+    "not include draft milestones" in {
```

Comment:
I would change this description to be "not include milestones for draft statements" - to me, this sounds like milestones can be drafts within an otherwise public statement.